src/main/java/com/checkmarx/flow/config/FlowConfig.java

```diff
@@ -66,6 +68,26 @@ public JavaMailSender getJavaMailSender() {
         return mailSender;
     }
 
+    public TemplateEngine getTemplateEngine() {
+
+        TemplateEngine templateEngine = new TemplateEngine();
+        templateEngine.setTemplateResolver(fileTemplateResolver());
+
+        return templateEngine;
+
+    }
+
+    public FileTemplateResolver fileTemplateResolver() {
+        FileTemplateResolver fileTemplateResolver = new FileTemplateResolver();
+        fileTemplateResolver.setSuffix(".html");
+        fileTemplateResolver.setTemplateMode(TemplateMode.HTML);
+        fileTemplateResolver.setCharacterEncoding("UTF-8");
```

Comment:
java.nio.charset.StandardCharsets.UTF_8 can be used instead

Reply:
Done